inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -354,5 +391,12 @@ air-electrolyte-reactions:
   exchange-current-density-formulation: true
   beta: 0.5
 
```

Comment:
Unless I am missing it, here, you will also eventually need a reaction along the lines of:

"LiO2(e) + Li(e) + electron <=> Li2O2"

I think you'll need one for both the conductor surface and catalyst surface.

But it might be easier for now not to worry about the catalyst.  Let's just get it up and running, with Carbon...?

Reply:
I'm not sure if this refers to the Main or HNG branch, but my understanding was that this Li2O2 reaction would not be in the Yaml file for HNG because its handled by the HNG reaction kinetics

Reply:
Good point.  This PR is to the `main` branch, where my understanding is we are first trying to just recreate the model w/o HNG.  For that implementation, we do want this in the YAML file, for now.